metadata/src/main/java/com/redhat/lightblue/metadata/parser/MetadataParser.java

```diff
@@ -181,6 +182,16 @@ public MetadataParser(Extensions<T> ex, TypeResolver typeResolver) {
         return this.extensions;
     }
 
+    private static Set<String> asSet(String... x) {
+        HashSet<String> set=new HashSet<>();
+        for(String a:x)
+            set.add(a);
```

Comment:
My personal preference is to use brackets even for one line loops and condition. Not a blocker to merging.